Examples/ObjectiveC/ShapeCollectionFeatureExample.m

```diff
@@ -0,0 +1,66 @@
+#import "ShapeCollectionFeatureExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleShapeCollectionFeature = @"ShapeCollectionFeatureExample";
+
+@interface ShapeCollectionFeatureExample () <MGLMapViewDelegate>
+@property (nonatomic) MGLMapView *mapView;
+@end
+
+@implementation ShapeCollectionFeatureExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURLWithVersion:9]];
```

Comment:
I think that for now, keeping the version number is safest on our end, with a disclaimer like "Please see the [Styles API documentation](https://www.mapbox.com/api-documentation/#styles) or the current version number" either in the example or on the examples page.

In the future, as @friedbunny has suggested elsewhere, it would be worth looking at how to automate version number updates within examples.